### PR TITLE
[Dotenv] extending env var list with debug:dotenv command

### PR DESCRIPTION
**Pull request by chr-hertel**
Was not sure about where to put this ...

https://github.com/symfony/symfony/pull/42580

### CODE REVIEW

File: configuration.rst [hunk 1]
```diff
@@ -718,8 +718,37 @@ you can encrypt the value using the :doc:`secrets management system </configurat
 Listing Environment Variables
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Regardless of how you set environment variables, you can see a full list with
-their values by running:
+To understand how environment variables are set by the Dotenv component parsing various
+files, you can run following command:
```
Comment (OskarStark) [inline configuration.rst]:
```suggestion
files, you can run the following command:
```

File: configuration.rst [hunk 2]
```diff
@@ -718,8 +718,37 @@ you can encrypt the value using the :doc:`secrets management system </configurat
...
+    ---------
+
+    ---------- ------- ---------- ------
+     Variable   Value   .env.dev   .env
+    ---------- ------- ---------- ------
+     FOO        BAR     n/a        BAR
+     ALICE      BOB     BOB        bob
+    ---------- ------- ---------- ------
+
+Additionally, and regardless of how you set environment variabls, you can see all
```
Comment (OskarStark) [inline configuration.rst]:
```suggestion
Additionally, and regardless of how you set environment variables, you can see all
```

**Comment by ro0NL** [comment]
see also https://symfony.com/doc/current/console/commands_as_services.html for new `is_enabled` it maybe introduces.

**Comment by javiereguiluz** [comment]
Christopher, thanks for providing this nice feature and also its docs!

**Pull request merged**